### ISSUE TITLE
Importing fails in renderer file.

### ISSUE THREAD

**Issue by abelta** [zoubingwu/electron-shared-state#11]
First time using the library here and trying to follow the tutorial.
My project is a Forge project in case that's relevant (shouldn't be).

Wasn't managing to make it work so I tried reducing my code to the minimum expression. The mere import of electron-shared-state in the renderer.js file makes the application not be able to build.

Countless errors appear on the shelf terminal but none of them seem descriptive (lots of mentions of path and fs but who knows if that's accurate).

In short:
// **renderer.js**
`import { createSharedStore } from 'electron-shared-state'`

This single line is taken from the tutorial and it makes the application crash.

Importing from **main.js** seems to work.
